drud/ddev-ui
# electron, and package update #115

This is **ONLY** a package update. Mainly updating Electron to a secure release above 1.8. Also updated the babel packages.

**To Test:**
1. Download the new build - https://376-99947022-gh.circle-artifacts.com/0/artifacts/DDEV%20UI-0.4.0-alpha.dmg
2. Make sure the app opens
3. Make sure you site list is the same as `ddev list`
4. Create a new site or connect an existing site
5. Start or stop an existing site in the list
